### PR TITLE
Mapbook v1.1

### PR DESCRIPTION
**Pull request by esreli**
This PR represents code that will make it version 1.1 of Offline Mapbook for iOS.

Enhancements Include
1. Couples login/logout UX with access to a portal. 
2. Removes confusion surrounding navigation bar iconography
3. Removes confusion surrounding switching app modes
4. Removes the ability to `.notSet` app mode in favor of an app mode default.
5. Retrieves last session app mode from UserDefaults.
6. Improves enums
7. Improves download did complete table refresh (now instant)
8. Leverages newer iOS navigation bar design, large titles.

This PR must ensure:

- [ ] Proper execution of enhancements
- [ ] Proper comments
- [ ] Updated README
- [ ] Project is configured properly

We also want to consider timing of the merge to master with publishing documentation to the developers site.

### CODE REVIEW

**Comment by esreli** [comment]
@jf990 Mapbook V1.1 has been approved both in code and documentation. At this point I can merge the change into master but want your take on whether it should wait to merge, coordinating with the developer site update, or if it's OK to go live now?

File: mapbook-iOS/Extensions/UIColor+House.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
+//
```
Comment (jf990) [inline mapbook-iOS/Extensions/UIColor+House.swift]: why name this extension `House` and not `primaryBlue` ?

Comment (esreli) [reply mapbook-iOS/Extensions/UIColor+House.swift]: It's named "House" as reference colloquially to "House Colors", meaning if we wanted to add more we could do so in the same extension.

Want me to change it?

Comment (jf990) [reply mapbook-iOS/Extensions/UIColor+House.swift]: I approved, I was just commenting to understand the justification it's fine by me so just leave it for now

**Comment by jf990** [comment]
@esreli this week's deploy already left the building, so we can get this in for next week. its ready to merge so let's move forward and get this in.

**Pull request merged**